libraries/Microsoft.Bot.Builder.Ai/QnAMaker.cs

```diff
@@ -76,6 +88,18 @@ public class QnAMakerOptions
         public string KnowledgeBaseId { get; set; }
         public float ScoreThreshold { get; set; }
         public int Top { get; set; }
+        public Metadata[] StrictFilters { get; set; }
+        public Metadata[] MetadataBoost { get; set; }
+    }
+
+    [Serializable]
```

Comment:
Please add a JSON Serialization Test around this, just to make sure the outputted JSON is of the right format.